### PR TITLE
F/#170 move cards to container

### LINKED ISSUE #170: [GLOBAL] Move Cards to a Container folder instead of elements
- [ ]  move it to container
- [ ] update the design system
- [ ] make sure the app running

### PR DESCRIPTION
**Pull request by f3ltron**
### PR check list
- [x] The title of the PR is formatted following [conventional commits](https://www.conventionalcommits.org/) format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Functionality has been verified and is ready to be code reviewed

### Relevant issues

Closes #170 

## What is the new behavior?

We are trying to implement https://github.com/viljamis/vue-design-system/wiki. So to as cards act like containers we should have containers folder and cards inside.

Paths are updated

**Does this PR introduce a breaking change?** 

No

### CODE REVIEW

File: components/containers/cards/cardevent.stories.js [hunk 1]
```diff
@@ -52,6 +52,6 @@ export const CardEvent = () => ({
 })
 
 export default {
-  title: 'Elements',
+  title: 'Containers',
```
Comment (T0RAT0RA) [inline components/containers/cards/cardevent.stories.js:55]: If we follow Atomic Design, shouldn't it be `Patterns`?

Comment (f3ltron) [reply components/containers/cards/cardevent.stories.js:55]: sure. I like the term container here but yeah i should update it to be consistent

Comment (T0RAT0RA) [reply components/containers/cards/cardevent.stories.js:55]: `Patterns/Containers`?

Comment (f3ltron) [reply components/containers/cards/cardevent.stories.js:55]: Stay consistent and let say patterns

**Pull request merged**